docs/example/README.md

```diff
@@ -106,7 +110,36 @@ In a browser, open up the following urls:
 
 Position the windows so that you can see all four at the same time. The header at the top of the webpage indicates the application and version.
 
-## Deploy SMI Access Control Policies
+## Traffic Policy Modes
+Once the applications are up and running, they can interact with each other using [SMI traffic policy mode](#smi-traffic-policy-mode) or [permissive mode](#permissive-traffic-policy-mode). In the SMI policy mode, all traffic is denied by default unless explicitly allowed using a combination of SMI access and routing policies. In permissive mode, traffic between application services is automatically configured by `osm-controller`, and SMI policies are not enforced.
+
+### Verify the Traffic Policy Mode
+Check whether permissive traffic policy mode is enabled or not by retrieving the value for the `permissive_traffic_policy_mode` key in the `osm-config` ConfigMap.
+```bash
+# Replace osm-system with osm-controller's namespace if using a non default namespace
+kubectl get configmap -n osm-system osm-config -o json | jq -r '.data["permissive_traffic_policy_mode"]'
+# Output:
+# false: permissive traffic policy mode is disabled, SMI policy mode is enabled
+# true: permissive traffic policy mode is enabled, SMI policy mode is disabled
+```
+
+The following sections demonstrate using OSM with SMI policies and permissive mode.
```

Comment:
Can we consistently reference the non permissive mode as either `SMI policy mode` or `SMI traffic policy mode`? Currently it's referenced both ways.

Reply:
Done.